components/AddressButtons/AddressButtons.tsx

```diff
@@ -12,39 +14,56 @@ const AddressButtons: React.FC<Props> = ({
   address,
   showInspectButton = true,
 }) => {
+  const { network } = useContext(NetworkContext);
+
+  const networkType = network.name === 'MAINNET' ? 'mainnet' : 'testnet';
+
+  const openProfileExplorer = () => {
+    window.open(
+      `https://wallet.universalprofile.cloud/${address}?network=${networkType}`,
+      '_blank',
+    );
+  };
+
+  const openAssetExplorer = () => {
+    window.open(
+      `https://wallet.universalprofile.cloud/asset/${address}?network=${networkType}`,
+      '_blank',
+    );
+  };
+
+  const openBlockscout = () => {
+    const url =
+      network.name === 'MAINNET'
+        ? `https://explorer.execution.mainnet.lukso.network/address/${address}`
+        : `https://explorer.execution.testnet.lukso.network/address/${address}`;
```

Comment:
you can `https://explorer.execution.${networkType}.lukso.network/address/${address}`
but either way is fine

Reply:
Good idea